src/kudu/consensus/log.cc

```diff
@@ -499,6 +505,16 @@ Log::Log(LogOptions options, FsManager* fs_manager, string log_path,
   }
 }
 
+Status LogFactory::createLog(LogOptions options,
+    FsManager* fs_manager, std::string log_path,
+    std::string tablet_id,
+    scoped_refptr<MetricEntity> metric_entity,
+    scoped_refptr<Log> *new_log) {
```

Comment:
pointer to a `scoped_refptr` as an out param?

Reply:
Yes, this is standard code pattern. All out parameters are pointers. Typical return value continues to remain a Status.